core/src/main/java/io/questdb/PropServerConfiguration.java

```diff
@@ -979,7 +979,7 @@ public PropServerConfiguration(
             this.o3LastPartitionMaxSplits = Math.max(1, getInt(properties, env, PropertyKey.CAIRO_O3_LAST_PARTITION_MAX_SPLITS, 20));
 
             // 1TB to disable by default
-            this.o3PartitionSplitMinSize = getLongSize(properties, env, PropertyKey.CAIRO_O3_PARTITION_SPLIT_MIN_SIZE, 1024L * 1024L * 1024L * 1024L);
+            this.o3PartitionSplitMinSize = getLongSize(properties, env, PropertyKey.CAIRO_O3_PARTITION_SPLIT_MIN_SIZE, 50 * Numbers.SIZE_1MB);
```

Comment:
do we keep this feature disabled until all bugs are fixed? Also value and comment diverged

Reply:
Done. The known squashing bug is now fixed on the master, we can enable splitting by default